projboard/models.py

```diff
@@ -52,6 +52,67 @@ def get_user_by_nickname(nickname):
             return None
         return user
 
+    @staticmethod
+    def create_user(email, password, name, nickname):
+        """
+         Method creates a new user
+        :param email: user's email
+        :param password: user's password
+        :param name: user's name
+        :param nickname: user's nickname
+        :return: new user
+        """
+        try:
+            user = User.objects.get(nickname=nickname)
+            user = User.objects.get(email=email)
```

Comment:
Why do we check for both nickname and email?  
isn't one check enough? (get user either by mail or by nickname)

Reply:
+1 here

Reply:
Done